src/hearings/services/hearings.service.ts

```diff
@@ -61,7 +61,7 @@ export class HearingsService {
     return this.http.put<HearingActualsMainModel>(`api/hearings/hearingActuals/${hearingId}`, hearingActualsModel);
   }
 
-  public submitHearingActuals(hearingId: string): Observable<number> {
-    return this.http.post<number>(`api/hearings/hearingActualsCompletion/${hearingId}`, null);
+  public submitHearingActuals(hearingId: string): Observable<HttpResponse<number>> {
```

Comment:
is this returning number?

Reply:
The post API does not return any data, but only HTTP status.

Reply:
res.status(status).send(data); I think the data is for this Observable<data:Object>, that is not for this Observable<status:number>, it will be Observable<void>more proper.